src/clj/orcpub/email.clj

```diff
@@ -30,15 +35,15 @@
    :host (environ/env :email-server-url)
    :port (Integer/parseInt (environ/env :email-server-port))})
 
-(defn send-verification-email [base-url {:keys [email username first-and-last-name]} verification-key]
+(defn send-verification-email [{:keys [email username first-and-last-name]} verification-key]
```

Comment:
You should not pull out base-url.

This is defining it as an argument to the function, which is called in the router src/clj/orcpub/routes.clj There, the request headers are being used to determine what the hostname is.

You should remove your environment variables, return this argument, and we should look into why it is getting the wrong request headers. I suspect it is either not a problem, just an artifact of some setup, or due to the nginx reverse proxy config munging the request headers.